Tests/WebApi.UnitTests/Applications/DirectorOperations/Commands/CreateDirector/CreateDirectorCommandTest.cs

```diff
@@ -0,0 +1,48 @@
+using AutoMapper;
+using FluentAssertions;
+using MovieStoreWebApi.Applications.DirectorOperations.Commands.CreateDirector;
+using MovieStoreWebApi.DbOperations;
+using MovieStoreWebApi.Entities;
+using Tests.WebApi.UnitTests.TestSetup;
+using Xunit;
+
+namespace Tests.WebApi.UnitTests.Applications.DirectorOperations.Commands.CreateDirector
+{
+    public class CreateDirectorCommandTest : IClassFixture<CommonTestFixture>
+    {
+        private readonly MovieStoreDbContext _context;
+        private IMapper _mapper;
+
+        public CreateDirectorCommandTest(CommonTestFixture textFixture)
+        {
+            _context = textFixture.Context;
+            _mapper = textFixture.Mapper;
+        }
+
+        [Fact]
+        public void WhenAlreadyExistActorIsGiven_InvalidOperationException_ShouldReturn()
+        {
+            var person = new Person()
+            {
+                Name = "Christopher",
+                Surname = "Nolan"
+            };
+            _context.Persons.Add(person);
+            _context.SaveChanges();
+            
+            var director = new Director()
+            {
+                PersonId = person.Id
+            };
+            _context.Directors.Add(director);
+            _context.SaveChanges();
+
+            CreateDirectorCommand command = new CreateDirectorCommand(_context, _mapper);
+            command.Model = new CreateDirectorModel() { Name = person.Name, Surname = person.Surname };
+
+            FluentActions.
+            Invoking(() => command.Handle()).Should().Throw<InvalidOperationException>()
+            .And.Message.Should().Be(CreateDirectorCommand.ExceptionMessage);
```

Comment:
Indetation'larla ilgili birden fazla yerde ayni seyi yapmissin, bunu unuttun sanirim ama bu sekilde olmasi biraz sagliksiz okuma acisindan. 
1- Uzunlugundan dolayi alt satira gecebilirsin ama 1 tab bosluk birakmalisin
2- O komut hala bitmemisse, yani ornegin 3 satirsa, her satirda bir bosluk birakilmaz, 2. ve 3. satirlar 1. satira gore 1 tab iceride olur, yani 2 ve 3 ayni hizada olurlar. Diger dosyalarda merdiven gibi yapmissin o da dogru degil.

Burada cok duzgun yazilmiyor ama ne demek istedigimi belli eder diye asagiya ornek olarak yaziyorum.
```suggestion
            FluentActions.Invoking(() => command.Handle())
                .Should().Throw<InvalidOperationException>()
                .And.Message.Should().Be(CreateDirectorCommand.ExceptionMessage);
```

Reply:
Düzeltildi.